djstripe/models.py

```diff
@@ -50,7 +51,9 @@
 
 @class_doc_inherit
 class Charge(StripeCharge):
-    __doc__ = getattr(StripeCharge, "__doc__")
+    """A model extending StripeCharge providing dj-stripe functionality."""
```

Comment:
https://github.com/PyCQA/pydocstyle/pull/203#issuecomment-243757226

Definitely check out https://pypi.python.org/pypi/flake8-docstrings/, since we're using flake8 already